synapse/federation/federation_client.py

```diff
@@ -266,6 +266,9 @@ def get_pdu(self, destinations, event_id, outlier=False, timeout=None):
             except NotRetryingDestination as e:
                 logger.info(e.message)
                 continue
+            except FederationDeniedError as e:
+                logger.debug(e.message)
```

Comment:
Is there a reason to log these at debug rather than info? Debug logging is normally disabled, so I'm worried that people will  get obscure failure modes due to lack of logging

Reply:
this was a thinko which i forgot to fully fix - thanks.